iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/FunctionDeclarationImplTest.java

```diff
@@ -32,23 +33,26 @@
 class FunctionDeclarationImplTest extends BicepTreeModelTest {
 
   @Test
-  void shouldParseFunctionDelaration() {
+  void shouldParseFunctionDeclaration() {
     ArmAssertions.assertThat(BicepLexicalGrammar.FUNCTION_DECLARATION)
-      .matches("func myFunction lambdaExpression")
-      .matches("func myFunction   lambdaExpression")
+      .matches("func myFunction() string => 'result'")
```

Comment:
I was a bit worried hat `myFunction()` could also be interpreted as a `functionCall `instead of a mixture of `IDENTIFIER `and `typedLambaExpression`, but thankfully we don't mix `typedLambaExpression `with `functionCall`.